src/tools/gophish_import.py

```diff
@@ -202,29 +208,33 @@ def build_campaigns(api, assessment):
                 break
             except Error as e:
                 if e.message == "SMTP name already in use":
-                    logging.warning(f"{e}. Finding previously loaded smtp to delete.")
+                    logging.warning("%s. Finding previously loaded smtp to delete.", e)
                     smtps = api.smtp.get()
                     logging.debug(
-                        f"Checking {len(smtps)} for previously imported smtp profiles to get ID"
+                        "Checking %i for previously imported smtp profiles to get ID",
+                        len(smtps),
                     )
                     for old_smtp in smtps:
                         if old_smtp.name == new_smtp.name:
-                            logging.debug(f"Deleting SMTP with ID {old_smtp.id}")
+                            logging.debug("Deleting SMTP with ID %i", old_smtp.id)
                             api.smtp.delete(old_smtp.id)
                             logging.info("Re-Loading new SMTP.")
                 else:
-                    logging.error(f"{e}\n")
+                    logging.exception(
+                        "Exception encountered when loading SMTP in Gophish (%s)",
+                        e.message,
+                    )
                     raise
 
         # Check to remove any campaigns with the same name
         old_campaigns = api.campaigns.get()
         for old_campaign in old_campaigns:
             if old_campaign.name == campaign["name"]:
                 logging.warning(
-                    f"Previous Campaign found with name {campaign['name']}."
+                    "Previous Campaign found with name %s.", campaign["name"]
                 )
                 logging.warning(
-                    f"Previous Campaign with id {old_campaign.id} being deleted."
+                    "Previous Campaign with id %i being deleted.", old_campaign.id
```

Comment:
Same question as above about `%d` or `%s`:
```suggestion
                    "Previous Campaign with id %d being deleted.", old_campaign.id
```

Reply:
See aa93585